src/transformers/convert_slow_tokenizer.py

```diff
@@ -622,6 +622,17 @@ def decoder(self, replacement, add_prefix_space):
     def converted(self) -> Tokenizer:
         tokenizer = self.tokenizer(self.proto)
 
+        # Add user defined symbols
+        user_defined_symbols = [
+            AddedToken(token, normalized=False, special=False)
+            for token in [p.piece for p in self.proto.pieces if p.type == 4]
```

Comment:
let's add a comment that references the documentation of sentencepiece as to why we use 4 (arbitrary number)